bitnami/concourse/templates/worker/service.yaml

```diff
@@ -24,6 +21,6 @@ spec:
   ## ref: https://concourse-ci.org/internals.html#component-tsa
   ##
   ports: []
-  selector:
-    app.kubernetes.io/component: worker
+  {{- $podLabels := merge .Values.worker.podLabels .Values.commonLabels }}
+  selector: {{- include "common.labels.matchLabels" ( dict "customLabels" $podLabels "context" $ ) | nindent 4 }}
```

Comment:
I think you removed the `app.kubernetes.io/component: worker` label selector by mistake.

Reply:
fixed at https://github.com/bitnami/charts/pull/18460/commits/7dcb2715a4fd5050c9d9342f981c53814299ef77